frontend/src/app/sessions/session/session.component.html

```diff
@@ -99,7 +102,12 @@
       <div [ngClass]="draggingActive ? 'iframe-overlay' : ''"></div>
       <iframe
         [id]="'session-' + session.id"
-        class="w-full height"
+        [ngClass]="{
+          'w-full': true,
+          'heigh-fullscreen': fullscreenService.isFullscreen$ | async,
+          height: (fullscreenService.isFullscreen$ | async) === false
+        }"
```

Comment:
```suggestion
```

1. `w-full` is defined twice (here and in the `class` attribute).
2. Neither `height` nor `heigh-fullscreen` are needed here. The height should be handled by the element which contains the session title bar and the iframe.

Reply:
I completely reworked the styling and tried to remove as much unnecessary css as possible. I think its best to just have another look over it.